### PR TITLE
Kubernetes Support

### PR DESCRIPTION
**Pull request by tamas-molnar**

### CODE REVIEW

File: helm/generic-rw-s3/app-configs/concepts-rw-s3_publishing.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Values used for the deployed application.
+replicaCount: 1
+service:
+  name: concepts-rw-s3
+bucketName: com.ft.upp.normalised-source-concepts.k8s-pub
```
Comment (tamas-molnar) [inline helm/generic-rw-s3/app-configs/concepts-rw-s3_publishing.yaml]: Should be `upp-concept-normalised-store-k8s-eu`

File: helm/generic-rw-s3/app-configs/generic-rw-s3_delivery.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Values used for the deployed application.
+replicaCount: 1
+service:
+  name: generic-rw-s3
+bucketName: com.ft.upp.normalised-source-concepts.k8s-pub
```
Comment (tamas-molnar) [inline helm/generic-rw-s3/app-configs/generic-rw-s3_delivery.yaml]: should be `com.ft.universalpublishing.v2-suggestions-cache.k8s.test`

**Comment by peteclark-ft** [comment]
See https://github.com/Financial-Times/k8s-aws-delivery-poc/pull/155

**Comment by tamas-molnar** [comment]
I can't approve my own PR but consider it approved ✅

File: Dockerfile [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
+  && REPOSITORY="repository=$(git config --get remote.origin.url)" \
+  && REVISION="revision=$(git rev-parse HEAD)" \
+  && BUILDER="builder=$(go version)" \
+  && LDFLAGS="-X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
+  && echo "Fetching dependencies..." \
+  && go get -u github.com/kardianos/govendor \
+  && $GOPATH/bin/govendor sync \
+  && echo "Building app..." \
+  && echo "Build flags: $LDFLAGS" \
+  && CGO_ENABLED=0 go build -a -installsuffix cgo -ldflags="${LDFLAGS}" -o /${PROJECT} ${REPO_PATH} \
```
Comment (nwrigh) [inline Dockerfile]: I haven't seen this before ( CGO_ENABLED=0 or installsuffix) but the build-info is no longer working in k8s:
{
    "version": "Version  is not a semantic version",
    "repository": "Repository  does not match regex ((git|ssh|http(s)?)|(git@[\\w\\.]+))(:(//)?)([\\w\\.@\\:/\\-~]+)(\\.git)?(/)?",
    "revision": "Revision  does not match regex ^[0-9a-f]{5,40}$",
    "builder": "",
    "dateTime": "dateTime  does not match regex ^[0-9]{14}"
}

**Pull request merged**